intellij/workspace/setup/.idea/eclipseCodeFormatter.xml

```diff
@@ -4,7 +4,7 @@
     <option name="projectSpecificProfile">
       <ProjectSpecificProfile>
         <option name="eclipseVersion" value="CUSTOM" />
-        <option name="formatter" value="ECLIPSE" />
+        <!-- <option name="formatter" value="ECLIPSE" /> -->
```

Comment:
we can disable the plugin (set active to `false`) here:
https://github.com/devonfw/ide-settings/blob/d1a7dd1f1ae817b3fb659f92a8fb54eb567b3b74/intellij/plugins/EclipseCodeFormatter.properties#L2

Then you can delete this file.

Reply:
@hohwille see my first note:
* Eclipse had to be disabled in` eclipseCodeFormatter.xml`. Setting `plugin_active=false` in `intellij\plugins\EclipseCodeFormatter.properties` wasn't working for me.

That's why I came with the other solution 😃. Don't know if I was doing something wrong.

Or which file do you mean could be deleted?